core/src/View/MDSpan/Kokkos_MDSpan_Accessor.hpp

```diff
@@ -388,6 +388,23 @@ class ReferenceCountedAccessor {
   NestedAccessor m_nested_acc;
 };
 
+template <class ElementType, class MemorySpace>
+using checked_reference_counted_accessor =
```

Comment:
snake_case here is somewhat inconsistent with the rest of the code base
(we typically would use PascalCase for such an alias of a class template)

Reply:
renamed